test/parallel/test-stream-readable-async-iterators.js

```diff
@@ -375,11 +375,25 @@ async function tests() {
     // eslint-disable-next-line no-unused-vars
     for await (const a of r) {
     }
-    // eslint-disable-next-line no-unused-vars
-    for await (const b of r) {
+    try {
+      // eslint-disable-next-line no-unused-vars
+      for await (const b of r) {
+      }
+    } catch (err) {
+      assert.strictEqual(err.code, 'ERR_STREAM_ITERATOR_EXISTS');
```

Comment:
This test would have to be split in two. The previous code verified that `iterating on an ended stream completes` - this would have to be tested without the first `for await`. Then , we should add a new test about the `ERR_STREAM_ITERATOR_EXISTS` error.